src/js/pre-need/containers/ConfirmationPage.jsx

```diff
@@ -14,29 +15,83 @@ const scrollToTop = () => {
 };
 
 class ConfirmationPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isExpanded: false };
+  }
+
   componentDidMount() {
     focusElement('.confirmation-page-title');
     scrollToTop();
   }
 
+  toggleExpanded = (e) => {
```

Comment:
I think you can remove this and the isExpanded state. Doesn't look like it's used here.

Reply:
Thanks @jbalboni, removed. Should I do the same for the burial and pension confirmation pages?